pallets/subtensor/src/migration.rs

```diff
@@ -474,3 +488,48 @@ pub fn migrate_to_v2_fixed_total_stake<T: Config>() -> Weight {
         Weight::zero()
     }
 }
+
+pub fn migrate_to_v7_weight_commits<T: Config>() -> Weight {
+    let mut weight = T::DbWeight::get().reads(1);
+    let current_version = Pallet::<T>::on_chain_storage_version();
+
+    if current_version < 7 {
+        log::info!(
+            target: "runtime::subtensor",
+            "Migrating WeightCommits to new format (v7)"
+        );
+
+        // Iterate through all entries in the old storage
+        for (netuid, account, old_value) in OldWeightCommits::<T>::iter() {
+            weight = weight.saturating_add(T::DbWeight::get().reads(1));
```

Comment:
get the total weight based on the count of map. not accumulate it each time.

Reply:
done